subsidy/views.py

```diff
@@ -17,15 +17,26 @@ def default(self, obj):
 
 
 def subsidy_create(request):
+    if request.user.is_anonymous:
+        form= CreateNewSubsidy()
+    else:
+        form = CreateNewSubsidy(initial={'ong': request.user.ong})
     if request.method == "POST":
         form = CreateNewSubsidy(request.POST)
+
         if form.is_valid():
-            form.save()
-            return redirect("/subsidy/list")
+            ong=request.user.ong
+            subsidy=form.save()
+            subsidy.ong=ong
+            subsidy.save()
+            
 
+            return redirect("/subsidy/list")
+        else:
+            messages.error(request, 'Formulario con errores')
+          
 
-    form = CreateNewSubsidy()
-    return render(request, 'subsidy_form.html', {"form": form, "title": "Crear Subvención"})
+    return render(request, 'subsidy/create.html', {"form": form,"object_name":"subvención" })
```

Comment:
You missed the `title` here and the window shows `localhost:8000/subsidy/create`

Reply:
That explains lot of things.